Resume/Work.dhall

```diff
@@ -8,8 +8,6 @@
     , endDate : Optional Text
     , summary : Optional Text
     , highlights : Optional (List Text)
-    , skills : Optional (List Text)
-    , salaries : Optional (List (./Salary.dhall).Type)
```

Comment:
Could you please not remove these types, I know they are not part of the JSON Resume schema but I find the relevant and they are totally optional.

Reply:
Done. Even though I would like to keep this schema-compliant, I understand your desire to keep these fields. I have a reorganization proposal that would allow both variants to co-exist, but I'll propose that as a separate PR. For the scope of this PR, I have reverted the removal of these fields.

Reply:
Great, can't wait to see what's coming!